cdap-ui/app/features/streams/streams.less

```diff
@@ -40,5 +40,25 @@ body.state-streams-detail {
     }
 
   }
-
+  &.theme-cdap {
+    // Explorer view
+    .table.table-explore {
+      margin-top: 0;
+      border-right: 1px solid fade(@cdap-bluegray, 40%);
+      border-bottom: 1px solid fade(@cdap-bluegray, 40%);
+      th, td {
+        border-left: 1px solid fade(@cdap-bluegray, 40%);
+         &:hover { cursor: default; }
+      }
+      thead {
+        background-color: @cdap-header;
+        th { color: white; }
+      }
+      tbody {
+        background-color: #DCDCE4;
+        color: @cdap-header;
+      }
+      td { border-top: 1px solid fade(@cdap-bluegray, 40%); }
+    }
+  }
```

Comment:
Was initially in theme.less, but Ajai and I moved this style block to both datasets.less and streams.less since everything contained applies specifically to the accordion table styles under the Exlplore tab.